src/molecules/gv-row-expandable.js

```diff
@@ -14,50 +14,72 @@
  * limitations under the License.
  */
 import { css, html, LitElement } from 'lit-element';
+import { link } from '../styles/link';
+import { classMap } from 'lit-html/directives/class-map';
+import { dispatchCustomEvent } from '../lib/events';
 
 /**
  * Expandable row
  *
+ * @fires gv-expandable-row:toggle - When user toggle row
+ *
  * @attr {Boolean} open - When true, opens details by default
+ * @attr {String} icon - An optional icon to show open feature
+ * @attr {String} icon-opened -  An optional icon to show close feature
+ * @attr {Boolean} only-summary - When true, only summary is clickable for toggle row
```

Comment:
`only-summary` isn't straight forward to understand maybe we should rename it with something with the opposite meaning: Collapse on details clicked --- When false, only summary is clickable to collapse the row

But I didn't find an easy name 😞